src/_guides/language/sound-faq.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: guide
+title: "Sound Dart FAQ"
+description: "Questions and answers on how and why to write sound Dart code."
+toc: false
+---
+
+This page collects some questions and answers about why and how to
+write sound Dart code. Be sure to also check out the
+[Sound Dart Guide](/guides/language/sound-dart) to learn more.
```

Comment:
Could we add a screenshot to make this page more digestible and visual? Something like a screenshot of an IDE (or DartPad) with a strong mode error underlined.